commands/Blink/main.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/stianeikeland/go-rpio/v4"
+)
+
+func main() {
+	// level := byte(1)
+	// r := raspi.NewAdaptor()
+	// ledPwm := gpio.NewDirectPinDriver(r, "12")
+	// ledEncoderA := gpio.NewDirectPinDriver(r, "22")
+	// ledEncoderB := gpio.NewDirectPinDriver(r, "24")
+	// ledDir := gpio.NewDirectPinDriver(r, "26")
+	// ledBrake := gpio.NewDirectPinDriver(r, "32")
+
+	pinPwm := rpio.Pin(18)
```

Comment:
proof with two libraries. let doc, let examples and the implemented pid with layesr